thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -296,13 +312,16 @@ export default Route.extend(AuthenticatedRouteMixin, {
       session,
       anomalyId,
       anomalyUrn,
-      anomalyEntity,
+      anomalyTemplateEntity,
       contextUrnsPredefined,
       selectedUrnsPredefined,
-      anomalyUrnsPredefined,
-      metricTemplate
+      anomalyUrnsPredefined
     } = model;
 
+    // The metric 
```

Comment:
Could you put a comment here on why we do this?

Reply:
Added comment.